test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -89,7 +91,8 @@ public class CompactRecordLogger {
           entry("DECISION_REQUIREMENTS", "DRG"),
           entry("EVALUATION", "EVAL"),
           entry("SIGNAL_SUBSCRIPTION", "SIG_SUBSCRIPTION"),
-          entry("SIGNAL", "SIG"));
+          entry("SIGNAL", "SIG"),
+          entry("COMMAND_DISTRIBUTION", "CMD_DSTR"));
```

Comment:
🔧 Smaller is better. Let's use smallest for the default.

As this replaces the Deployment Distribution concept, let's abbreviate this to `DSTR` and abbreviate Deployment Distributions to `DPLY_DSTR`

This would also highlight when unexpected things happen (somehow Deployment Distribution is used somewhere unexpectedly).